tools/Dockerfile.ubuntu18.04.install

```diff
@@ -0,0 +1,57 @@
+# ==============================================================================
+#  Copyright 2019 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+
+ARG base_image
+
+FROM $base_image as base
+ARG http_proxy
+ARG https_proxy
+ARG ngtf_build_options
+
+RUN git clone https://github.com/tensorflow/ngraph-bridge.git /ngraph-bridge
+
+WORKDIR /ngraph-bridge
+
+ENV http_proxy=$http_proxy
+ENV https_proxy=$https_proxy
+
+RUN echo "Using build options: $ngtf_build_options" && ./build_ngtf.py $ngtf_build_options
+
+FROM ubuntu:18.04
+
+ARG http_proxy
+ARG https_proxy
+
+ENV http_proxy=$http_proxy
+ENV https_proxy=$https_proxy
+
+ENV PATH="/ngraph-bridge/build_cmake/venv-tf-py3/bin:$PATH"
+
+RUN apt-get update &&  apt-get install -y \
+    vim \
+    python3-distutils \
+    python3-pip \
+    git \
+    unzip zip wget \
```

Comment:
split these all into one per line, also sort the packages by name.
We also use these switches when possible:
```
apt-get install -y --no-install-recommends --fix-missing
```